src/login/logind.c

```diff
@@ -88,6 +88,8 @@ static Manager *manager_new(void) {
         if (!m->kill_exclude_users)
                 goto fail;
 
+        m->kill_user_processes = true;
+
```

Comment:
Not sure how I feel about this one. It's quite a change, we so far didn't dare to make.

I am actually OK with making this change upstream, but I figure we need to make this a "noisy" change, with a larger addition to NEWS, and a clear message to distros what's at stake here, how they can revert to old behaviour, and what our recommended way to handle screen/tmux is.

Moreover, this probably should be something that gets a "configure" switch for selecting the default, where we default to on.

(And on the Fedora side I figure this needs a feature page...)